src/Boards.h

```diff
@@ -123,7 +123,7 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 // #define TOTAL_PINS              21 //defined in spark_wiring.h
 // #define TOTAL_ANALOG_PINS       8 //defined in spark_wiring.h
 #define VERSION_BLINK_PIN       7
```

Comment:
change 7 to `LED_BUILTIN` per https://github.com/spark/firmware/blob/develop/wiring/inc/spark_wiring_arduino.h#L160